blocks_cli_setup.php

```diff
@@ -0,0 +1,77 @@
+<?php
```

Comment:
can you please move this file under /cli/blocks_cli_setup.php ?

Reply:
The file has been moved under cli/